winml/lib/Api/LearningModelBinding.h

```diff
@@ -60,7 +61,7 @@ struct LearningModelBinding : LearningModelBindingT<LearningModelBinding, ILearn
 
   std::unordered_map<std::string, ProviderInfo> m_providers;
 
-  std::unique_ptr<onnxruntime::IOBinding> m_lotusBinding;
+  std::unique_ptr<_winmla::IOBinding> m_lotusBinding;
```

Comment:
If I'm understanding this correctly, by making this a smartpointer std::unique_ptr, the _winmla::IOBinding's descructor will be called when LearningModelBinding is destructed right?

Reply:
yup !

---
In reply to: [345525728](https://github.com/microsoft/onnxruntime/pull/2382#discussion_r345525728) [](ancestors = 345525728)